extra-files/galaxy-kickstart/galaxy-kickstart_tool_list.yml

```diff
@@ -0,0 +1,37 @@
+tools:
+- name: data_manager_fetch_genome_dbkeys_all_fasta
+  owner: devteam
+  revisions:
+  - 776bb1b478a0
+  tool_panel_section_label: null
```

Comment:
> No tool_panel_section_label at all, or a tool_panel_section_label for data managers ?

I think a data_managers tool panel section label would be good, even if it will never appear in the toolbox. I don't like the null too much.